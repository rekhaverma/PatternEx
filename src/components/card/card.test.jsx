import React from 'react';
import Card from './card.component';

/**
 * @status: Complete
 * @sign-off-by: Alex Andries
 */
describe('<Card />', () => {
  it('should match with snapshot', () => {
    const props = {
      children: <div />,
      footer: <div />,
      count: 'count',
      countValue: 1,
    };
    const tree = shallow(<Card {...props} />);
    expect(tree).toMatchSnapshot();
  });
});
