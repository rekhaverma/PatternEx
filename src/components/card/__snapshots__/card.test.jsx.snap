// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> should match with snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card__container"
  >
    <CardHeader
      count="count"
      countValue={1}
      fullWidth={false}
      hasHelp={false}
      hasMore={false}
      hasMoreValue={false}
      onHelpClick={[Function]}
      onMoreClick={[Function]}
      title="Untitled"
    >
      <div />
    </CardHeader>
    <div
      className="card__content"
    >
      <div />
    </div>
    <CardFooter
      content={<div />}
    />
  </div>
</div>
`;
